### PR TITLE
FullTextFields config parameter removal

### PR DESCRIPTION
**Pull request by nablaone**
This PR: 
- `FullTextFields` config parameter  is removed 
- schema type is used to determine which field is a 'full text' searchable

### CODE REVIEW

**Comment by pdelewski** [comment]
@nablaone To clarify, Am I correct that now, all fields that are `Strings` and not `keywords` will be treaded as `fulltext` fields?

File: quesma/schema/registry.go [hunk 1]
```diff
@@ -150,7 +150,7 @@ func (s *schemaRegistry) populateSchemaFromTableDefinition(definitions map[strin
 					fields[propertyName] = Field{PropertyName: propertyName, InternalPropertyName: FieldName(column.Name), Type: quesmaType}
 				} else {
 					logger.Debug().Msgf("type %s not supported, falling back to text", column.Type)
```
Comment (avelanarius) [inline quesma/schema/registry.go:152]:
```suggestion
					logger.Debug().Msgf("type %s not supported, falling back to keyword", column.Type)
```

Comment (avelanarius) [reply quesma/schema/registry.go:152]: (per change 1 line below)

**Review by mieciu — APPROVED** [review]
One thing @avelanarius caught, other than that LGTM :)

**Pull request merged**